Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/EfficiencyCondition.mo

```diff
@@ -0,0 +1,341 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.SetPoints.Subsequences;
+block EfficiencyCondition
+  "Efficiency condition used in staging up and down"
+
+  parameter Integer nSta = 5
+    "Number of stages in the boiler plant";
+
+  parameter Real fraNonConBoi(
+    final min=0,
+    final unit="1",
+    final displayUnit="1") = 0.9
```

Comment:
Just a reminder to remove `final`. Now that the code is formatted nicely, a simple bulk rename in a text editor can be used.